zerver/webhooks/gosquared/doc.md

```diff
@@ -1,23 +1,38 @@
+# Zulip GoSquared integration
+
 Receive GoSquared notifications in Zulip!
 
+{start_tabs}
+
 1. {!create-channel.md!}
 
 1. {!create-an-incoming-webhook.md!}
 
-1. {!generate-integration-url.md!}
+1. {!generate-webhook-url-basic.md!}
 
 1. Go to your project's **Settings** and click on **Services**.
    Scroll down and next to **Webhook**, click on **Connect**. Click
-   on **Add new**.
-
-1. Set **Webhook URL** to the URL constructed above. Set **Name** to
-   a name of your choice, such as `Zulip`. Click on **Save Integration**.
+   on **Add new**. Set **Webhook URL** to the URL constructed above.
```

Comment:
Also, we can use "generated" instead of "constructed" for the URL, across all the docs, to reflect the change that URLs are now generated.

Reply:
Good point!

Reply:
There are more occurrences of the same, in the other docs :)